template/_config.js

```diff
@@ -0,0 +1,5 @@
+System.config({
```

Comment:
Why a `config.js` instead of a `webpack.config.js`? 🤔 (See https://twitter.com/TheAlexLichter/status/1082945828270964736)

Reply:
I don't know why, but webpack.config.js didn`t work for me. So I prefer a solution which was successful for me.

Reply:
I think it is better to elaborate more on this. I was also able to use webpack.config for IntelliJ idea and should consider part (probably a lot) of users are using different IDEs so should consider that to have maximum coverage :)

Reply:
I think that we can't cover all possible IDE's. This PR was primary for IntelliJ so in first, you could merge solution for that. If people with different IDE will face this problem, they have a chance to do the same thing -> Design solution, create PR and enjoy the result.